### PR TITLE
Support PAY_PER_REQUEST billing mode

### PR DESCRIPTION
**Pull request by JakeChampion**
When using tables which are billed with "PAY_PER_REQUEST", the returned write/read capacity units is 0, which is not a valid unit to use when creating a table. write/read capacity units are not required for tables which are billed "PAY_PER_REQUEST".

### CODE REVIEW

File: src/helper.js [hunk 1]
```diff
@@ -356,6 +361,8 @@ const createGlobalDynamodbTable = async function createGlobalDynamodbTable(serve
       return
     }
 
+    cli.consoleLog(`CreateGlobalTable: ${chalk.yellow('Global Table configuration: ' + JSON.stringify(globalTablesOptions))}`);
```
Comment (JakeChampion) [inline src/helper.js]:
```suggestion
```

File: src/helper.js [hunk 2]
```diff
@@ -356,6 +361,8 @@ const createGlobalDynamodbTable = async function createGlobalDynamodbTable(serve
...
+
```
Comment (JakeChampion) [inline src/helper.js]:
```suggestion
```

File: src/helper.js [hunk 3]
```diff
@@ -414,7 +419,7 @@ const createGlobalDynamodbTable = async function createGlobalDynamodbTable(serve
       }
     }
   } catch (error) {
-    serverless.cli.consoleLog(`CreateGlobalTable: ${chalk.red(`Failed to setup global table. Error ${error.message || error}`)}`)
+    serverless.cli.consoleLog(`CreateGlobalTable: ${chalk.red(`Failed to setup global table. Error ${error}`)}`)
```
Comment (JakeChampion) [inline src/helper.js]:
```suggestion
    serverless.cli.consoleLog(`CreateGlobalTable: ${chalk.red(`Failed to setup global table. Error ${error.message || error}`)}`)
```

**Comment by rrahul963** [comment]
@JakeChampion - can you please update the package.json file with the version and also fix the unit test

**Pull request closed without merging**